package.json

```diff
@@ -57,6 +57,7 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
+    "handlebars": "^4.3.0",
```

Comment:
why is handlebars needed?
Also its in resolutions below, doesn't a yarn upgrade fix any audit issues? (its not been a problem in anything else)